Browser/CefSharpBrowser/CefOp/GadgetUrlHandler.cs

```diff
@@ -1,15 +1,30 @@
-﻿using CefSharp;
+﻿using BrowserLibCore;
+using CefSharp;
 using CefSharp.Handler;
+using IBrowser = CefSharp.IBrowser;
 
 namespace Browser.CefSharpBrowser.CefOp;
 
 internal class GadgetUrlHandler : ResourceRequestHandler
 {
+	private GadgetServerOptions GadgetBypassServer { get; }
+	private string GadgetBypassServerCustom {get; }
+	public GadgetUrlHandler(GadgetServerOptions gadgetBypassServer, string gadgetBypassServerCustom)
+	{
+		GadgetBypassServer = gadgetBypassServer;
+		GadgetBypassServerCustom = gadgetBypassServerCustom;
+	}
 	protected override IResponseFilter GetResourceResponseFilter(IWebBrowser chromiumWebBrowser, IBrowser browser, IFrame frame, IRequest request, IResponse response)
 	{
 		if (request.Url.Contains(@"gadget_html5"))
 		{
-			return new GadgetReplaceFilter("http://203.104.209.7/gadget_html5/", "https://kcwiki.github.io/cache/gadget_html5/");
+			return GadgetBypassServer switch
```

Comment:
Make an extension method for the gadget logic so we don't have these urls hardcoded in 2 places.
```cs
public static GadgetReplaceFilter GetReplaceFilter(this GadgetServerOptions option) => option switch
{
	...
};
```